app/javascript/components/collections/landing/Pagination.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+class Pagination extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleClick = (page, event) => {
+    event.preventDefault();
+    this.props.search.setState({ currentPage: page });
```

Comment:
I'm not liking calling `setState()` on a parent component from a `prop`....hmmm.  This could probably be better structured in a more conventional React way.   Like calling a callback function passed in as a `prop`, then in the parent component let it call it's own `setState()`.  Otherwise this could be confusing on where a component's state is being updated.